PhysicsTools/PatAlgos/python/slimming/MicroEventContent_cff.py

```diff
@@ -94,3 +94,50 @@
         'keep GenRunInfoProduct_*_*_*',
         'keep L1GtTriggerMenuLite_l1GtTriggerMenuLite__*',
 ]
+
+MiniAODOverrideBranchesSplitLevel = cms.untracked.VPSet( [
+cms.untracked.PSet(branch = cms.untracked.string("patMuons_slimmedMuons__*"),splitLevel=cms.untracked.int32(0)),
+cms.untracked.PSet(branch = cms.untracked.string("patElectrons_slimmedElectrons__*"),splitLevel=cms.untracked.int32(0)),
+cms.untracked.PSet(branch = cms.untracked.string("patTaus_slimmedTaus__*"),splitLevel=cms.untracked.int32(0)),
```

Comment:
is the list of branches here substantially different from what's in ```MicroEventContent``` ?
Are there any branches that have to stay in the full split or is the real intent of the changes here to make everything in miniAOD to be splitLevel 0?

It looks like most of the miniAOD branches in this PR are set to use the no-split mode.
It seems more practical to use the output module splitLevel parameter to set the default and then override a few branches to have the full split (like packedCandidates),